doc/development/tutorials/builders.rst

```diff
@@ -0,0 +1,32 @@
+Configuring builders
```

Comment:
Can you move this up a level? It's not really a tutorial, though it is development stuff